proto/rig/model/pipeline.proto

```diff
@@ -0,0 +1,38 @@
+syntax = "proto3";
+
+package model;
+
+import "google/protobuf/timestamp.proto";
+import "google/protobuf/duration.proto";
+
+message Pipeline {
+  // Custom name for the pipeline.
+  string name = 1;
+  repeated Phase phases = 2;
+  google.protobuf.Timestamp created_at = 3;
+  google.protobuf.Timestamp updated_at = 4;
+}
+
+message Phase {
+  // Environment ID. The project must be active in this environment.
+  string environment_id = 1;
+  // Fixed fields, that are not changed upon promotion.
+  repeated string fixed_fields = 2;
```

Comment:
Do we use the new json path stuff here as the field identifiers?

Reply:
That is the intention